includes/models/class-reviews.php

```diff
@@ -149,8 +181,10 @@ public function dismiss_review() {
 	/**
 	 * Update user meta.
 	 *
-	 * @param int   $user_id - the user id.
-	 * @param array $review - the review.
+	 * @param int   $user_id. The user id.
```

Comment:
We should remove the period after `$user_id`. The period after `The user id` is correct.

Please do the same for other @param and @return lines as well.